Test.Utility/MockContainerBuilder.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Autofac;
+using Moq;
+
+namespace Messerli.Test.Utility
+{
+    public static class MockContainerBuilder
```

Comment:
Rename class to something with ´Extension´ in the name i. E ´ContainerBuilderMockExtension´

Reply:
Addressed in: https://github.com/messerli-informatik-ag/test-utility/pull/20/commits/4d33a0c474230aebf9a54c1d4d870ecb7c356783